js/levels.js

```diff
@@ -241,7 +241,7 @@ var levels = [
       'tr': '<p>Grid çizgilerinin başlangıç ve bitiş konumlarına dayalı olarak bir grid öğesi tanımlamak yerine, <code>span</code> anahtar kelimesini kullanarak, onu istediğiniz sütun genişliğine göre tanımlayabilirsiniz. <code>span</code> özelliğinin sadece pozitif değerler ile çalıştığını unutmayın.</p><p>Örneğin, havuçları bu kural ile sulamayı deneyin: <code>grid-column-end: span 2;</code>.</p>',
       'fr': '<p>Au lieu de définir un élément de la grille basé sur les positions de début et de fin des lignes de la grille, vous pouvez le définir en fonction de la largeur de colonne désirée à l\'aide du mot clé <code>span</code>. Gardez à l\'esprit que <code>span</code> fonctionne uniquement avec des valeurs positives.</p><p>Par exemple, arrosez ces carottes avec la règle <code>grid-column-end: span 2;</code>.</p>',
       'de': '<p>Anstatt ein Rasterelement basierend auf Start- und Endposition der Rasterlinien zu definieren, kannst du es anhand der gewünschten Spaltenbreite mit dem Schlüsselwort <code>span</code> definieren. Denk daran, dass <code>span</code> nur mit positiven Werten funktioniert.</p><p>Wässere zum Beispiel diese Karotten mit der Regel <code>grid-column-end: span 2;</code>.</p>',
-      'nl': '<p>Inplaats van een grid item te definiëren op basis van de start en eind posities van de grid lijnen, kan je hem definiëren op basis van het aantal colommen dat je wilt invullen, door het <code>span</code> sleutelwoord te gebruiken. Onthoud dat <code>span</code> alleen werkt met positieve waardes.</p><p>Bijvoorbeeld, geef deze planten water met de regel  <code>grid-column-end: span 2;</code>.</p>',
+      'nl': '<p>I.p.v. een grid item te definiëren op basis van de start- en eindposities van de gridlijnen, kan je hem definiëren op basis van het aantal kolommen dat je wilt invullen, door het <code>span</code>-sleutelwoord te gebruiken. Onthoud dat <code>span</code> alleen werkt met positieve waardes.</p><p>Geef bijv. deze planten water met de regel <code>grid-column-end: span 2;</code>.</p>',
```

Comment:
Using abbreviations makes reading more restless. I think it's clearer with the words written out.